tasks/server-0.11.x.yml

```diff
@@ -69,19 +81,44 @@
   run_once: true
   register: migration
 
-- debug: var=migration.stdout_lines
-  when: migration|success
+- name: Copy over kong init script
+  template:
+    src: kong.init.j2
+    dest: "/etc/init.d/kong"
+    owner: root
+    group: root
+    mode:  0744
+  when: kong_manage_init_script
+
+- name: Create sysvinit symlinks
+  service:
+    name: kong
+    enabled: yes
+  when: kong_manage_init_script
```

Comment:
I think this should be a separate task - we want kong to be enabled on startup, even if it's currently running, to ensure it enables kong-on-boot for our existing instances, not just new ones